fast/stages/3-network-security/README.md

```diff
@@ -69,7 +69,7 @@ ln -s ~/fast-config/tfvars/2-networking.auto.tfvars.json ./
 
 # copy and paste the following commands for '3-network-security'
 
-gcloud storage cp gs://xxx-prod-iac-core-outputs-0/providers/3-netsec-providers.tf ./
+gcloud storage cp gs://xxx-prod-iac-core-outputs-0/providers/3-nsec-providers.tf ./
```

Comment:
I'd keep the providers matching the stage name, or also rename the stage to nsec.